invenio_openaire/loaders.py

```diff
@@ -211,6 +215,46 @@ def grantxml2json(self, grant_xml):
         }
         return ret_json
 
+    def grantjson2json(self, grant_json):
+        """Convert OpenAIRE grant JSON into comptaible JSON."""
+        funder_doi = 0  # data is available, just have to be decided how it is fetched depending on the final integration
+        
+        to_replace = 0  # Code used to generate eurepo identifier
+
+        grant_id = grant_json["code"]
+        id_ = f"{funder_doi}::{grant_id}"
+
+        ret_json = {
+            "created": datetime.isoformat(datetime.now()),
+            "id": id_,
+            "links": {
+                "self": f"https://zenodo.org/api/grants/{id_}"
+            },
+            "metadata": {
+                "$schema": "http://zenodo.org/schemas/grants/grant-v1.0.0.json",
+                "acronym": grant_json.get("acronym", ""),
+                "code": grant_id,
+                "enddate": grant_json.get("enddate", ""),
+                "funder": {
+                    "$ref": f"http://dx.doi.org/{funder_doi}"
+                },
+                "identifiers": {
+                    "eurepo": 0,  # f"info:eu-repo/grantAgreement/{grant_json["funding"][0]["shortName"]}/{to_replace}/{grant["code"]}/",
```

Comment:
```suggestion
                    "eurepo": "info:eu-repo/grantAgreement/{shortName}/{program}/{code}/",
```

From the other loader apparently we can use the `program` field (or empty string if it doesn't exist) for the middle part.